Telegram/SourceFiles/platform/win/notifications_manager_win.cpp

```diff
@@ -608,8 +608,8 @@ void queryQuietHours() {
 		return;
 	}
 
-	LPTSTR lpKeyName = L"Software\\Microsoft\\Windows\\CurrentVersion\\Notifications\\Settings";
-	LPTSTR lpValueName = L"NOC_GLOBAL_SETTING_TOASTS_ENABLED";
+	LPCWSTR lpKeyName = L"Software\\Microsoft\\Windows\\CurrentVersion\\Notifications\\Settings";
+	LPCWSTR lpValueName = L"NOC_GLOBAL_SETTING_TOASTS_ENABLED";
```

Comment:
As long as we're using `RegOpenKeyEx` instead of `RegOpenKeyExW` — yes. Either use `RegOpenKeyExW` with `LPCWSTR` or use `RegOpenKeyEx` with `LPCTSTR`.